### ISSUE TITLE
Building this plugin requires the amazon-util jar from teamcity-aws-codedeploy-plugin project

### ISSUE THREAD

**Issue by rodm** [JetBrains/teamcity-aws-codepipeline-plugin#2]
Checking out and building the plugin fails due to a dependency on the amazon-util.jar from the 
[code deploy plugin](https://github.com/JetBrains/teamcity-aws-codedeploy-plugin). Shoudn't the amazon-util code be extracted into its own project?

**Comment by vbedrosova** [comment]
Yes, we do plan to extract the amazon-util.jar library but now it's borrowed from the aws-codedeploy-plugin as you've noticed
